hms-app/src/common/types.tsx

```diff
@@ -1,20 +1,50 @@
+export type HackathonPreview = {
+  id: string
+  title: string
+}
+
+export type IdeaPreviews = {
+  id: string
+  title: string
+}
+
 export type Idea = {
   id: string
+  owner: Owner[]
+  hackathon: HackathonPreview[]
+  participants: Participant[]
   title: string
   description: string
-  author?: User
-  reason: string
   problem: string
   goal: string
-  skills: string[]
-  category: string
-  participants: User[]
-  aws: boolean
+  requiredSkills: Skill[]
+  category: Category[]
```

Comment:
same here, will always just be a single value not an array

Reply:
sure about that? dont we get an array with skill objects?